ISISNeutronMuon/MDANSE
# Multiple window fix and triple panel update

**Description of work**
Fixed multiple window opening for the toolbar windows. Updated the Converters and Actions tabs so that it uses a triple panel layout. The action dialog has been removed and the actions are now in the 3rd panel on the right.

![image](https://github.com/ISISNeutronMuon/MDANSE/assets/14276033/bf0fa9eb-9ca4-4fa0-98ad-8f7b9a3b298f)

**Fixes**
closes #302

**To test**
Triple panel tests
1. On the converter tab clicking any of the converters should change the action panel.
2. Clicking on things which are not a converter (e.g. Materials Studio) should give a blank panel.
3. Converting a trajectory should work as normal.
4. Without a trajectory loaded go to the action tabs and click some of the analysis. The action panel should be blank since there are no trajectories selected.
5. Load a trajectory in the trajectory tab.
6. Go to the actions tab and check that actions panel is now populated.
7. Clicking on other analysis jobs should change the actions panel.
8. Load another trajectory and try changing the trajectory in the actions tab. The actions panel should update.
9. Running a job and loading the plot should work as usual.
10. Try deleting all trajectories in the 3D view. The trajectory and the actions panel should update accordingly. 

Toolbar window tests
- Test the toolbar windows. They should only open once now and bring the window back to the foreground when minimised and etc.

Note that there is a ongoing issue with trajectory deletion #319 from the trajectory tab and trajectory loading from the 3D view #307. Please be aware of this while testing.
